### ISSUE TITLE
Slider na stronie głównej

### ISSUE THREAD

**Issue by 9orky** [szpak-dev/szpak-dev.github.io#29]

**Comment by 9orky** [comment]
@TomekJantos Wszystko ok poza jednym plikiem: `Plano All Weather.jpg` Nazwa pliku odpowiada chyba całej kategorii / serii a nie konkretnemu produktowi.

**Comment by TomekJantos** [comment]
Tak, jak najbardziej odnośnikiem może być link do całej kategorii
produktowej.

Czy można zobaczyć jak to teraz wygląda?
W sensie po włączeniu slajdera?

Pozdrawiam / Best regards

Tomek Jantos
***@***.***
www.greyhunter.com.pl
GREY HUNTER FACEBOOK PAGE
<https://www.facebook.com/profile.php?id=100090934454415>
mob. +48 502770556

Grey Hunter Tomasz Jantos
ul. Belgradzka 18/108 02-793 Warszawa
NIP: 9512111440 REGON: 146876673




czw., 11 maj 2023 o 14:21 Tomasz Szpak ***@***.***>
napisał(a):

**Comment by 9orky** [comment]
Problem polega na tym, że w Slajderze linkujemy tylko do konkretnych produktów, skoro eksponujemy w nim produkty ;) 
Tak więc jakiś konkretny trzeba by wybrać. 

PS: Jak Slajder będzie gotowy, to dam ci od razu znać.

**Comment by TomekJantos** [comment]
Ok, wobec tego niech przekierowuje na stronę innej walizki all weather -
11842

Pozdrawiam / Best regards

Tomek Jantos
***@***.***
www.greyhunter.com.pl
GREY HUNTER FACEBOOK PAGE
<https://www.facebook.com/profile.php?id=100090934454415>
mob. +48 502770556

Grey Hunter Tomasz Jantos
ul. Belgradzka 18/108 02-793 Warszawa
NIP: 9512111440 REGON: 146876673




czw., 11 maj 2023 o 15:28 Tomasz Szpak ***@***.***>
napisał(a):

**Comment by 9orky** [comment]
Slajder zrobiony, zapraszam popatrzeć :)

**Comment by TomekJantos** [comment]
Ok, trzy pytania,

- dlaczego jest inne zdjęcie Plano? a tego brakuje?
 Plano All Weather.jpg
<https://drive.google.com/file/d/1N76g5NGO9UQhSS2gT5mtis0xRS4aCpqM/view?usp=drive_web>
Chodziło mi o to żeby je wstawić tylko żeby odnośnik link czy jak go
zwał... po kliknięciu w to zdjęcie przenosi nas do produktu 11842 i ju.

- czy te zdjęcia mogą być na całą szerokość ekranu? bo mi wyświetlają się z
paskami po boku
- teksty nałożone na zdjęcia na slajderze: większe, białą czcionką,
kapitalikami, przezroczyste w sensie bez tła jak na stronie Marttiini,
zawsze wyświetlane w jednym z narożników np. lewy dół.
RAKKA 2023
LYNX 131
ALL WEATHER 11852
ALL WEATHER 11842

Pozdrawiam / Best regards

Tomek Jantos
***@***.***
www.greyhunter.com.pl
GREY HUNTER FACEBOOK PAGE
<https://www.facebook.com/profile.php?id=100090934454415>
mob. +48 502770556

Grey Hunter Tomasz Jantos
ul. Belgradzka 18/108 02-793 Warszawa
NIP: 9512111440 REGON: 146876673




czw., 11 maj 2023 o 16:56 Tomasz Szpak ***@***.***>
napisał(a):

**Comment by 9orky** [comment]
> Ok, trzy pytania, - dlaczego jest inne zdjęcie Plano? a tego brakuje? Plano All Weather.jpg <https://drive.google.com/file/d/1N76g5NGO9UQhSS2gT5mtis0xRS4aCpqM/view?usp=drive_web> Chodziło mi o to żeby je wstawić tylko żeby odnośnik link czy jak go zwał... po kliknięciu w to zdjęcie przenosi nas do produktu 11842 i ju. 

Nie mam uprawnień do powyższego linku do Google Drive, ale podejrzewam, że chodzi Ci zdjęcie faceta ładującego walizkę na czarnego melexa? Czeski błąd, zaraz będzie właściwe.

> - czy te zdjęcia mogą być na całą szerokość ekranu? bo mi wyświetlają się z paskami po boku

Oczywista

> - teksty nałożone na zdjęcia na slajderze: większe, białą czcionką, kapitalikami, przezroczyste w sensie bez tła jak na stronie Marttiini, zawsze wyświetlane w jednym z narożników np. lewy dół. RAKKA 2023 LYNX 131 ALL WEATHER 11852 ALL WEATHER 11842

Może być biały font, ale co z fotkami, które mają białe tło? Wtedy nic nie będzie widać :) A Marttiini głównie ma takie. No chyba, że na slajder zawsze będziemy mieli takie fotki, że biały będzie widoczny.

 Pozdrawiam / Best regards Tomek Jantos ***@***.*** [www.greyhunter.com.pl](http://www.greyhunter.com.pl) GREY HUNTER FACEBOOK PAGE <https://www.facebook.com/profile.php?id=100090934454415> mob. +48 502770556 Grey Hunter Tomasz Jantos ul. Belgradzka 18/108 02-793 Warszawa NIP: 9512111440 REGON: 146876673 czw., 11 maj 2023 o 16:56 Tomasz Szpak ***@***.***> napisał(a):

**Comment by TomekJantos** [comment]
Tak, właśnie to zdjęcie z tym kolesiem! Super

Na sjalderze planuje zdjęcia raczej z tłem tam gdzie produkt będzie
widoczny w jakimś kontekście (natury, strzelnicy, polowania, domu).  Więc z
dużym prawdopodobieństwem zawsze biały tekst będzie się odcinał.

Pozdrawiam / Best regards

Tomek Jantos
***@***.***
www.greyhunter.com.pl
GREY HUNTER FACEBOOK PAGE
<https://www.facebook.com/profile.php?id=100090934454415>
mob. +48 502770556

Grey Hunter Tomasz Jantos
ul. Belgradzka 18/108 02-793 Warszawa
NIP: 9512111440 REGON: 146876673




pt., 12 maj 2023 o 12:32 Tomasz Szpak ***@***.***> napisał(a):
